docs/CHANGELOG-1.11.md

```diff
@@ -0,0 +1,643 @@
+## Parity [v1.11.7](https://github.com/paritytech/parity/releases/tag/v1.11.7) (2018-07-16)
```

Comment:
Can we highlight couple of important changes to transaction queue that happened in 1.11?
Namely:
- The pool now limits transactions per-sender (see --tx-queue-per-sender), local transactions also have to obey that limit. Consider increasing the limit via CLI flag when running benchmarks or sending a lot of transactions at once.

- In case the pool is full transactions received over the network, but originating from accounts that we have private keys for might not get accepted to the pool any more with higher priority. Consider running with larger pool size or submitting the transactions directly on the node via `eth_sendRawTransaction`.